### PR TITLE
fix regression introduced to check object store

### PR DESCRIPTION
**Pull request by camilamacedo86**
#### What this PR does / why we need it:

Fix regression introduced within release `v2023.03.20-0` to verify the object store.

**Error:**
Screenshot 2023-03-21 at 07 55 08

**Scenario**:
- :white_check_mark: (Works) With the release v2023.03.13-0
- Install: curl https://kurl.sh/version/v2023.03.13-0/4d8927d | sudo bash -s yes
- Upgrade: curl https://kurl.sh/version/v2023.03.13-0/d39b1bb | sudo bash -s yes

- :red_circle: (Fails with latest)
- Install: curl https://kurl.sh/4d8927d | sudo bash -s yes
- Upgrade: curl https://kurl.sh/d39b1bb | sudo bash -s yes 

#### Which issue(s) this PR fixes:

Fixes # [sc-72502]

## Steps to reproduce
- Install: curl https://kurl.sh/4d8927d | sudo bash -s yes
- Upgrade: curl https://kurl.sh/d39b1bb | sudo bash -s yes

See the error fixed:

Screenshot 2023-03-21 at 11 24 39

Also, you can check the Object Store checks:

Screenshot 2023-03-21 at 11 24 27

- You can check the whole logs: [pr-logs.txt](https://github.com/replicatedhq/kURL/files/11027726/pr-logs.txt)

#### Does this PR introduce a user-facing change?

```release-note
Fixes bug regression introduced in the previous release `v2023.03.20-0` where the registry addon fails to create the object store.  
```

#### Does this PR require documentation?
NONE

### CODE REVIEW

**Comment by rrpolanco** [comment]
@camilamacedo86 It's not apparent to me what the issue was. What was the actual error encountered?

**Comment by camilamacedo86** [comment]
@rrpolanco,

It is all described in : sc-72502 Adding the following info in the description as well.

Error:
Screenshot 2023-03-21 at 07 55 08

See:
- :white_check_mark: (Works) With the release v2023.03.13-0
curl https://kurl.sh/version/v2023.03.13-0/4d8927d | sudo bash -s yes
curl https://kurl.sh/version/v2023.03.13-0/d39b1bb | sudo bash -s yes

- :red_circle: (Fails with latest)
curl https://kurl.sh/4d8927d | sudo bash -s yes
curl https://kurl.sh/d39b1bb | sudo bash -s yes

**Comment by camilamacedo86** [comment]
@rrpolanco,

I create the shortcut described in the description of this PR which has all details [sc-72502]. 
Also, see that I added more details in the description of this PR either.

File: scripts/common/object_store.sh [hunk 1]
```diff
@@ -241,22 +241,17 @@ function migrate_rgw_to_minio_checks() {
...
+    if ! spinner_until 300 rook_rgw_check_if_is_healthy ; then
         logFail "Failed to detect healthy Rook Ceph Object Store"
         bail "Cannot upgrade from Rook Ceph Object Store to Minio. Rook Ceph is unhealthy."
     fi
 
     logSuccess "Rook Ceph Object Store to Minio migration checks completed successfully."
 }
 
-function rook_rgw_is_healthy() {
-    export OBJECT_STORE_CLUSTER_IP
```
Comment (ricardomaraschini) [inline scripts/common/object_store.sh:253]: IIUC the problem was that we were overwriting the `OBJECT_STORE_CLUSTER_IP`, is that correct ?

**Pull request merged**